integration-tests/src/test/java/oracle/weblogic/kubernetes/utils/DbUtils.java

```diff
@@ -719,4 +689,331 @@ public static void updateRcuAccessSecret(String secretName, String namespace,
 
   }
 
+  /**
+   * Install Oracle Database Operator.
+   *
+   * @param namespace name of the namespace
+   * @throws IOException when fails to modify operator yaml file
+   */
+  public static void installDBOperator(String namespace) throws IOException {
+    String operatorYamlFile = DOWNLOAD_DIR + "/oracle-database-operator.yaml";
+    String certManager = "https://github.com/jetstack/cert-manager/releases/latest/download/cert-manager.yaml";
+    CommandParams params = new CommandParams().defaults();
+    params.command("kubectl apply -f " + certManager);
+    boolean response = Command.withParams(params).execute();
+    assertTrue(response, "Failed to install cert manager");
+
+    assertDoesNotThrow(() -> TimeUnit.SECONDS.sleep(60));
+
+    String operatorYamlUrl = "https://raw.githubusercontent.com/"
```

Comment:
Can we move this hard-coded URL to TestConstant class

Reply:
done